lib/actions/push.ts

```diff
@@ -166,6 +176,14 @@ export const push: CommandDefinition<
 			description: stripIndent`
 				Path to a local YAML or JSON file containing Docker registry passwords used to pull base images`,
 		},
+		{
+			signature: 'live',
+			alias: 'l',
+			boolean: true,
+			description: stripIndent`
+				Start a live session after the push, which will wait for code changes, and synchronise them with
+				running containers. This mode is only valid when pushing to a local device.`,
```

Comment:
I suggest a short description here, with a reference, so it formats more nicely on the screen:
```
Start a live session after the push. See live push description above.
```